src/jestRunner.ts

```diff
@@ -50,7 +50,14 @@ export class JestRunner {
     await editor.document.save();
 
     const filePath = editor.document.fileName;
-    const command = this.buildJestCommand(filePath);
+    let command: string;
+
+    if (options) {
+      const testName = undefined;
+      command = this.buildJestCommand(filePath, testName, options);
+    } else {
+      command = this.buildJestCommand(filePath);
+    }
```

Comment:
Thanks for updating the pull request! I have another small change request:
Why the `if` statement? `options` is also an optional parameter in the `buildJestCommand()` function, so those 8 lines could be rewritten to

```
const command = this.buildJestCommand(filePath, undefined, options);
```

Reply:
Thanks for the feedback! I changed the code.